### PR TITLE
add a set_test_case_result command

### PR DESCRIPTION
**Pull request by elyamurza**

### CODE REVIEW

**Comment by elyamurza** [comment]
calls on bottom will be removed later, just keeping it for now for testing & how to determine documentation.

File: app.py [hunk 1]
```diff
@@ -45,20 +59,49 @@ def create_test_run(self, run_name=str, plan_id=str, suite_ids=[]):
...
+
+    def get_test_case_id(self, case_name):
+        cases = self.get_tests_in_run()
+        for c in cases:
+            if c['title'] == case_name:
+                return c['id']
+
+    """ Takes an optional value, can pass all_tests to pass through each test and give a passing score """
+
+    def set_test_case_result(self, case_name, result, *args):
```
Comment (debbiehwang) [inline app.py:79]: not sure this method is intuitive enough....
so pass in the case_name you want to pass, if you want to all tests to pass in the test run, you still need to pass in a case_name?

**Comment by debbiehwang** [comment]
will have to refactor the project structure if we want to make it a library (which is a good idea, to keep Receptionist simple)

File: app.py (hunk 1 above)
Comment (elyamurza) [reply app.py:79]: would it make sense to create a separate method that passes all test cases in a specific run? or somehow make it an optional variable

Comment (debbiehwang) [reply app.py:79]: yeah, or make `case_name=None` as default, so if user doesnt pass in case_name, it defaults to pass all tests ? does that make sense

Comment (elyamurza) [reply app.py:79]: Yep! makes sense, i'll work on a fix for that

Comment (elyamurza) [reply app.py]: @debbiehwang updated

**Pull request merged**